expr/queue.go

```diff
@@ -0,0 +1,68 @@
+package expr
+
+import (
+	"encoding/binary"
+
+	"github.com/google/nftables/binaryutil"
+	"github.com/mdlayher/netlink"
+	"golang.org/x/sys/unix"
+)
+
+type QueueAttribute uint16
+
+type QueueFlag uint16
+
+// Possible QueueAttribute values
+const (
+	QueueAttributeNum   QueueAttribute = unix.NFTA_QUEUE_NUM
```

Comment:
For consistency, these should just be `QueueNum`, `QueueTotal` and `QueueFlags` (without the `Attribute` part).

Reply:
It's done.